packages/js/email-editor/src/components/header/style.scss

```diff
@@ -10,3 +10,83 @@
 		display: none;
 	}
 }
+
+.woocommerce-email-editor__view-mode-toggle {
```

Comment:
Could you please share more details on why we need to use the custom styles and not just reuse styles for the post editor we already load?
I noticed a commit message about some problems with colors.
I ran into some problems with colors recently. The preset vars were not defined outside the editor canvas. I fixed the issue in [the PR with UI fixes](https://github.com/woocommerce/woocommerce/pull/58078) that was merged a few day ago. So perhaps if the problem was the same, we might not need the CSS 🙏

Reply:
I am testing it now, but with the default editor classes, the CSS does not load properly: https://d.pr/i/jwViee

I'm not sure what I'm missing, but that's why I used the custom styles. 

This is the screenshot from the site editor: https://d.pr/i/pcjyfB

Reply:
Aha. We load CSS for the post editor. There are still some differences. Could you please check the markup in the post editor? I see it [uses](https://github.com/WordPress/gutenberg/blob/f995b6c84d203002e4c7c4cc46e6a95204a1e0c9/packages/edit-post/src/components/back-button/fullscreen-mode-close.js#L79) `edit-post-fullscreen-mode-close` and `has-icon` classes. Maybe these could help.

Reply:
Ok. Let's stick with the custom CSS then.